packages/viewer/src/components/viewer-hit-result-indicator/viewer-hit-result-indicator.tsx

```diff
@@ -79,6 +79,8 @@ export class ViewerHitResultIndicator {
         .getPropertyValue('--viewer-hit-result-indicator-plane-opacity')
         .trim();
 
+      console.log(this.arrowColor);
```

Comment:
Nit: is this needed or can we remove it?

Reply:
Shoot, missed that being left in. Removed!